sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication.TestCommon/Constants.cs

```diff
@@ -49,6 +49,7 @@ public class Constants
         public static readonly string NotSupportedInNetCoreError = "not supported for .NET Core";
 
         // Connection strings
+        //[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Secret is used for tests only")]
```

Comment:
Should this be removed?

Reply:
We already have existing CredScan suppressions in the code, this is just accounting for a newer detection. I see some other SDKs do as well. Is there any specific guidance on this?

Reply:
I assumed that you commented the line out and I was suggesting to remove it. If this is required by a tool, then i'm fine with it.